pyigm/guis/igmguesses.py

```diff
@@ -1929,7 +1943,68 @@ def from_igmguesses_to_joebvp(infile, outfile):
 
     ltiu.joebvp_from_components(comp_list, igmg_dict['spec_file'], outfile)
 
+def init_meta():
+    """Creates a general meta dictionary
 
+    Returns
+    -------
+    meta : dict
+
+    """
+    meta = dict()
+    meta['RA_deg'] = 0.
+    meta['DEC_deg'] = 0.
+    meta['object'] = "unknown"
+    meta['zem'] = 0.
+    meta['instrument'] = 'unknown'
```

Comment:
How about we add the date automatically? Based on the date the .json file is written?